From: Cyril Brulebois <kibi@debian.org>
Date: Tue, 26 Dec 2006 12:24:24 +0100
Subject: Fix OS detection of darkplaces' makefile to not think it's on BSD on
 GNU/kFreeBSD

[The DarkPlaces build system mostly cares about userland, not the kernel,
and GNU/kFreeBSD is equivalent to GNU/Linux in that respect. -smcv]

Origin: vendor, Debian
Forwarded: no
---
 makefile | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/makefile b/makefile
index 5533f14..d18ba6a 100644
--- a/makefile
+++ b/makefile
@@ -9,6 +9,10 @@ else
 
 # UNIXes
 DP_ARCH:=$(shell uname)
+ifeq ($(DP_ARCH), GNU/kFreeBSD)
+	# same userspace as Linux, not a BSDish one
+	DP_MAKE_TARGET=linux
+else
 ifneq ($(filter %BSD,$(DP_ARCH)),)
 	DP_MAKE_TARGET=bsd
 else
@@ -23,6 +27,7 @@ else
 endif  # ifeq ($(DP_ARCH), SunOS)
 endif  # ifeq ($(DP_ARCH), Darwin)
 endif  # ifneq ($(filter %BSD,$(DP_ARCH)),)
+endif  # ifeq ($(DP_ARCH), GNU/kFreeBSD)
 endif  # ifdef windir
 endif  # ifndef DP_MAKE_TARGET
 
